packages/react-dogfood/hooks/useWakeLock.ts

```diff
@@ -0,0 +1,32 @@
+import { CallingState, useCallCallingState } from '@stream-io/video-react-sdk';
+import { useEffect } from 'react';
+
+interface WakeLockSentinel {
+  release(): Promise<void>;
+}
+
+export const useWakeLock = () => {
```

Comment:
Nice! 🚀 

Shall we maybe include this hook in the SDK? And, perhaps enable it by default in the default layouts?